exampleCourse/questions/template/checkbox/random-prompt/server.py

```diff
@@ -0,0 +1,27 @@
+import random
+
+import numpy as np
+
+
+def is_prime(a):
+    if a == 1:
+        return False
+    return all(a % i for i in np.arange(2, a))
```

Comment:
```suggestion
    return all(a % i for i in range(2, a))
```
No need to use the numpy range here, I think you can get rid of the import completely.

Reply:
Done in 7799f9a.